.gitignore

```diff
@@ -8,6 +8,8 @@
 # Dependency directories
 node_modules/
 
+.node-version
```

Comment:
I don't think this should be in the gitignore? It'll be useful for us to run on a consistent node version (if this file isn't here already, I have one in my branch set to `12.16.1` and can merge it shortly)

Reply:
will revove